app/controllers/items_controller.rb

```diff
@@ -23,6 +23,28 @@ def new
   def edit
   end
 
+  # GET /items/borrowed
+  # lists all items borrowed by the current user
+  def borrowed_items
```

Comment:
- style: IMO the "items" in this route name is redundant because the controller already implies that we are talking about items; suggestion to rename to, for example, "borrowed_by_me"